Content.Shared/MedicalScanner/SharedHealthAnalyzerComponent.cs

```diff
@@ -0,0 +1,48 @@
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.HealthAnalyzer
+{
+    public abstract class SharedHealthAnalyzerComponent : Component
+    {
+        [Serializable, NetSerializable]
+        public class HealthAnalyzerBoundUserInterfaceState : BoundUserInterfaceState
+        {
+            public readonly string? TargetName;
+            public readonly bool? IsAlive;
+            public readonly string? TotalDamage;
+            public readonly List<MobDamageGroup> DamageGroups;
+
+            public HealthAnalyzerBoundUserInterfaceState(
+                string? targetName,
+                bool? isAlive,
+                string? totalDamage,
+                List<MobDamageGroup> damageGroups)
+            {
+                TargetName = targetName;
+                IsAlive = isAlive;
+                TotalDamage = totalDamage;
+                DamageGroups = damageGroups;
+            }
+        }
```

Comment:
* Target name not needed.
* * IsAlive probably not needed and can be moved to shared.  
* Realistically you could probably just have this contain an EntityUid and then have the client handle it all. This also means you don't need the mobdamagegroup and it cuts down on the state's bandwidth a lot.

Reply:
Going to opt for this,  only passing in an entityuid and getting the damageable and name on the client.
The only downside is that there the info isn't saved between closing and opening the UI but it's definitely a lot cleaner.